modules/installation-localzone-generate-k8s-manifest.adoc

```diff
@@ -26,8 +26,57 @@ $ ./openshift-install create manifests --dir <installation_directory> <1>
 <1> For `<installation_directory>`, specify the installation directory that
 contains the `install-config.yaml` file you created.
 
-. Create the machine set manifests for the worker nodes in your Local Zone.
+. Set the default Maximum Transmission Unit (MTU) according to the network plugin:
++
+[IMPORTANT]
+====
+Generally, the Maximum Transmission Unit (MTU) between an Amazon EC2 instance in a Local Zone and an Amazon EC2 instance in the Region is 1300. See link:https://docs.aws.amazon.com/local-zones/latest/ug/how-local-zones-work.html[How Local Zones work] in the AWS documentation. 
+The Cluster network MTU must be always less than the EC2 MTU to account for the cluster network overlay overhead. The specific overhead is determined by your network plugin:
+
+- OVN-Kubernetes: 100 bytes
+- OpenShift SDN: 50 bytes
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../networking/changing-cluster-network-mtu.adoc#mtu-value-selection_changing-cluster-network-mtu[MTU value selection]
+====
+// Note: Steps described on the official doc https://docs.openshift.com/container-platform/4.12/networking/changing-cluster-network-mtu.html#mtu-value-selection_changing-cluster-network-mtu
```

Comment:
This line should be removed after we decided if we will make any reference to this page or not
```suggestion
```